.github/release.yml

```diff
@@ -7,6 +7,10 @@ changelog:
         labels:
           - dependencies
           - breaking_change
+          - add-new-address
+    - title: 🆕 New Chain Addresses
```

Comment:
I think this would be better below the rest of the changes, as this is not as relevant as the breaking changes

Reply:
Makes sense. Changed.